### PR TITLE
feat: add ml.preprocessing.MinMaxScaler

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: tests/system/small/ml/test_preprocessing.py [hunk 1]
```diff
@@ -211,6 +211,97 @@ def test_max_abs_scaler_series_normalizes(penguins_df_default_index, new_penguin
...
+    pd.testing.assert_frame_equal(result, expected, rtol=1e-3)
+
+
+def test_min_max_scaler_series_normalizes(penguins_df_default_index, new_penguins_df):
+    scaler = bigframes.ml.preprocessing.MinMaxScaler()
+    scaler.fit(penguins_df_default_index["culmen_length_mm"])
+
+    result = scaler.transform(penguins_df_default_index["culmen_length_mm"]).to_pandas()
+
+    # If maxabs-scaled correctly, max should be 1.0
```
Comment (GarrettWu) [inline tests/system/small/ml/test_preprocessing.py:244]: MinMax will have min 0 and max 1.

File: bigframes/ml/preprocessing.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def _parse_from_sql(cls, sql: str) -> tuple[MaxAbsScaler, str]:
             sql: SQL string of format "ML.MAX_ABS_SCALER({col_label}) OVER()"
 
         Returns:
-            tuple(StandardScaler, column_label)"""
+            tuple(MaxAbsScaler, column_label)"""
```
Comment (GarrettWu) [inline bigframes/ml/preprocessing.py:153]: Thanks!

File: tests/system/small/ml/test_preprocessing.py [hunk 3]
```diff
@@ -244,6 +244,8 @@ def test_min_max_scaler_series_normalizes(penguins_df_default_index, new_penguin
     # If minmax-scaled correctly, min should be 0 and max should be 1.
     for column in result.columns:
         assert math.isclose(result[column].max(), 1.0, abs_tol=1e-3)
+    for column in result.columns:
```
Comment (GarrettWu) [inline tests/system/small/ml/test_preprocessing.py:247]: nit: you can do both checks in the same loop.

**Review by GarrettWu — APPROVED** [review]

File: tests/system/small/ml/test_preprocessing.py (hunk 3 above)
Comment (ashleyxuu) [reply tests/system/small/ml/test_preprocessing.py]: Done

**Pull request merged**